example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# This is currently a placeholder field and must be left empty for now.
+Principals: []
 
+# Path to the SNS Project logo on the local filesystem. The path is relative
+# to the configuration file location, unless an absolute path is given.
+# Must have less than 341,334 characters, roughly 256 Kb. The only supported
+# format is PNG.
 logo: logo.png
+
+# URL to the dapp controlled by the SNS project.
+# Must be a string of max length = 512.
```

Comment:
To @mraszyk 's point: if there are constraints here, let's list them (see next comment)